tests/conftest.py

```diff
@@ -10,12 +10,12 @@
 # backends to be tested
 BACKENDS = [
     "tensorflow",
-    "pytorch",
-    "jax",
+    # "pytorch",
+    # "jax",
```

Comment:
```suggestion
    "pytorch",
    "jax",
```
reactivate these to check that `pytorch` and `jax` are still working